source/content/wordpress-launch-check.md

```diff
@@ -23,6 +23,24 @@ In short, you get a fast, repeatable report that can help detect common problems
 
 WP Launch Check is a site audit extension for WP-CLI designed for Pantheon customers. While designed initially for the Pantheon Dashboard it is intended to be fully usable outside of Pantheon.
 
+## WPScan Dependency
+
+WP Launch Check utilizes the [WPScan plugin](https://wordpress.org/plugins/wpscan/) to check for outdated or vulnerable plugins. If you wish to use this service to receive an alerts when your plugins need to be updated, you'll need to obtain an [API token](https://wpscan.com/pricing) from their website and configure your site to use your token.
+
+### Configuring WPScan
+
+Configuring WPScan is easy. Once you've obtain your API token, add it to your site's `wp-config.php` file using the following PHP code: 
+
+```php:title=wp-config.php
+define( 'WPSCAN_API_TOKEN', '$your_api_token' );
+```
+
+Alternatively, you could use the [Terminus Secrets Plugin](https://github.com/pantheon-systems/terminus-secrets-plugin) to add you API token at the platform level. The API token is [accessed from WP Launch Check](https://github.com/pantheon-systems/wp_launch_check/blob/9ddfd899b80ff42fa3ab060c2f44b930e8278069/php/pantheon/checks/plugins.php#L80) via `getenv('PANTHEON_WPVULNDB_API_TOKEN')`. To set this environment variable, follow the steps to install the Terminus Secrets Plugin, and add your token to your chosen environment:
+
+```bash
+terminus secrets:set <site>.<env> PANTHEON_WPVULNDB_API_TOKEN <value>
```

Comment:
I see references to `PANTHEON_WPVULNDB_API_TOKEN` here.  It would be nice to have the code look for a variable named `WPSCAN_API_TOKEN` so that the variable name can remain consistent throughout.

While we've not tried to keep the value of `PANTHEON_WPVULNDB_API_TOKEN` a secret, it's possible someone might find it, try to use it, and have a poor experience.

Reply:
I removed this blurb in [6d96d84](https://github.com/pantheon-systems/documentation/pull/7246/commits/6d96d8427036aec08a214195c4afdf4689b97070)